blivet/formats/fs.py

```diff
@@ -399,6 +415,12 @@ def _post_create(self, **kwargs):
                 self.write_label()
             except FSError as e:
                 log.warning("Failed to write label (%s) for filesystem %s: %s", self.label, self.type, e)
+        if self.new_uuid is not None and self.can_reassign_uuid():
+            try:
+                self.write_uuid()
+            except FSError as e:
+                log.warning("Failed to write UUID (%s) for filesystem %s: %s",
```

Comment:
Yeah, it makes even more sense to not log this in the first place but actually directly throw an exception.